tests/components/__snapshots__/do_verify_email.test.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/DoVerifyEmail should match snapshot 1`] = `
+<LoadingScreen
+  position="relative"
+/>
+`;
```

Comment:
Should add snapshots test for 
- when `state.verifyStatus` is  'pending'` and,
- when `state.serverError` is not null.